microsoft-edge/test-and-automation/test-and-automation.md

```diff
@@ -0,0 +1,86 @@
+---
+title: Test and automation in Microsoft Edge
+description: DevTools Protocol instruments, debugs, profiles browsers.  Origin Trials to try experimental APIs.  Playwright cross-browser automation via 1 API.  Puppeteer API automates via DevTools Protocol.  WebDriver simulates user interaction.  webhint linting checks for errors, best practices.
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.topic: conceptual
+ms.prod: microsoft-edge
+ms.technology: devtools
+ms.date: 02/14/2022
+---
+# Test and automation in Microsoft Edge
+
+There are many tools to automate your testing of Microsoft Edge:
+
+| Tool | Description |
+| --- | --- |
+| DevTools Protocol | Instrument, inspect, debug, and profile browsers including Microsoft Edge. |
+| Origin Trials | Try out experimental APIs on live sites for a limited period of time. |
+| Playwright | The Playwright library provides cross-browser automation through a single API. |
+| Puppeteer | The Puppeteer library provides a high-level API to control Microsoft Edge using the DevTools Protocol. |
```

Comment:
```suggestion
| Puppeteer | The Puppeteer library provides a high-level API to control browsers, including Microsoft Edge, using the DevTools Protocol. |
```

Reply:
Applied fix 3x: 
this summary table, 
the h2 below, and 
the Puppeteer article.